dev/import-beats-resources/nginx/docs/README.md

```diff
@@ -0,0 +1,80 @@
+# Nginx Integration
+
+This integration periodically fetches metrics from [https://nginx.org/](Nginx) servers.
+
+
+## Compatibility
+
+The Nginx stubstatus metrics were tested with Nginx 1.9 and are expected to work with all version >= 1.9. The logs were tested with version 1.10. On Windows, the module was tested with Nginx installed from the Chocolatey repository.
+
+
+## Logs
+
+
+**Timezone support**
+
+This datasource parses logs that don’t contain timezone information. For these logs, the Elastic Agent reads the local timezone and uses it when parsing to convert the timestamp to UTC. The timezone to be used for parsing is included in the event in the event.timezone field.
+
+To disable this conversion, the event.timezone field can be removed with the drop_fields processor.
+
+If logs are originated from systems or applications with a different timezone to the local one, the event.timezone field can be overwritten with the original timezone using the add_fields processor.
+
+### Access Logs
+
+Access logs collects the nginx access logs.
+
+{{fields "access"}}
+
+### Error Logs
+
+Error logs collects the nginx error logs.
+
+{{fields "error"}}
+
+## Metrics
+
+### Stubstatus Metrics
+
+The Nginx stubstatus stream collects data from the Nginx ngx_http_stub_status module. It scrapes the server status data from the web page generated by ngx_http_stub_status.
+
+This is a default stream. If the host datasource is unconfigured, this stream is enabled by default.
+
+An example event for nginx looks as following:
+
+```$json
+{
```

Comment:
Where did you take this even from? Existing from Beat? This will probably look slightly different when shipped through the agent.

Reply:
Yes, I will refresh the samples once I run the agent.

Reply:
https://github.com/elastic/package-registry/issues/317